.idea/misc.xml

```diff
@@ -35,7 +35,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
this file should never be updated, every IDE keeps modifying it randomly. I had the same problem with xan

Reply:
Can we not commit it to the repository then?

Reply:
In some places I read to ignore the entire .idea folder, (https://stackoverflow.com/questions/38590022/should-we-ignore-misc-xml-and-projectfoldername-iml-in-pycharm-idea-folder) in some others (https://stackoverflow.com/a/39570738/1187692) that it is needed by the IDE.

For now remove it from this commit

Reply:
Ok, done!